tests/test_generate_copy_without_render.py

```diff
@@ -68,3 +78,24 @@ def test_generate_copy_without_render_extensions():
 
     with open('test_copy_without_render/rendered/not_rendered.yml') as f:
         assert '{{cookiecutter.render_test}}' in f.read()
+
+    if overwrite_if_exists:
+
+        # This should in all cases:
+        generate.generate_files(
+            context=context,
+            repo_dir='tests/test-generate-copy-without-render',
+            overwrite_if_exists=overwrite_if_exists
+        )
+
+    else:
+
+        # This should fail, with OutputDirExistsException:
+        try:
+            generate.generate_files(
+                context=context,
+                repo_dir='tests/test-generate-copy-without-render',
+                overwrite_if_exists=overwrite_if_exists
+            )
+        except OutputDirExistsException:
+            pass
```

Comment:
```suggestion
        with pytest.raises(OutputDirExistsException):
            generate.generate_files(
                context=context,
                repo_dir='tests/test-generate-copy-without-render',
                overwrite_if_exists=overwrite_if_exists
            )
```

Reply:
@nicain, @kurtmckee Could you try to rebase this and make the tests pass? If not, I will close it

Reply:
@ericof I tried; the merge conflicts didn't seem easily surmountable without more effort.